platform/src/abstractions/EducationBenchmarking.Platform.Infrastructure/Search/SearchOptions.cs

```diff
@@ -9,19 +9,4 @@ public abstract class SearchServiceOptions
     
     public Uri Endpoint => new($"https://{ServiceName}.search.windows.net/");
     public AzureKeyCredential Credential => new(ApiKey);
-}
-
-public class TrustSearchServiceOptions : SearchServiceOptions
-{
-    public string TrustIndexName { get; set; }
-}
-
-public class SchoolSearchServiceOptions : SearchServiceOptions
-{
-    public string SchoolIndexName { get; set; }
-}
-
-public class LocalAuthoritySearchServiceOptions : SearchServiceOptions
-{
-    public string LocalAuthorityIndexName { get; set; }
 }
```

Comment:
Has this been removed as it has been abstracted in the datacollection class?

Reply:
nevermind! Found them.

Reply:
No, I just put it at the wrong level.